### DISCUSSION TITLE
Question About about:config option

### DISCUSSION THREAD

**Discussion by premalska** [Divested-Mobile/Mull-Fenix#262] [Q&A] [answered]
Because of vulnerability on 129.0.2, I updated my mull to 132.0.0
For my personal reason, I toggled some options in about:config

Which mean,
privacy.resistFingerprinting = false
privacy.fingerprintingProtection = true
privacy.fingerprintingProtection.overrides = +AllTargets,-FontVisibilityBaseSystem,-FontVisibilityLangPack,-FontVisibilityRestrictGenerics

I was using those options without any problem previous version

But, In 132.0.0 I cannot change "privacy.fingerprintingProtection" to "true"

More precisely, changes made to "true" are not persisted

If I toggle it to "true" and relaunch the browser or reboot my phone, the option will remain changed to "false"

Is there any way to resolve this issue?

**Comment by SkewedZeppelin** [comment, marked as answer]
fingerprintingProtection is controlled via ETP Strict, you'll have to turn that on via the regular settings (it is by default though).
I really suggest keeping RFP on as Mull intends especially to look like other Mull users.

**Reply by premalska** [reply]
Thanks for your answer
My ETP Setting was already Strict, but I briefly selected another option and reverted back to strict then it worked fine
